experimental/addons/chaosblade-operator/README.md

```diff
@@ -0,0 +1,139 @@
+
+# Chaosblade-operator: A Chaos Engineering Tool for Cloud-native
+
+[Chaosblade-operator](https://github.com/chaosblade-io/chaosblade-operator) chaosblade operator for kubernetes experiments
+                                  
+
+This addon based on the [Chaosblade-operator](https://github.com/chaosblade-io/chaosblade-operator). The addon itself will deploy chaosblade-operator for every application.
+
+
+## Install
+
+Add experimental registry
+```
+vela addon registry add experimental --type=helm --endpoint=https://addons.kubevela.net/experimental/
+```
+
+Enable this addon
+```
+vela addon enable chaosblade-operator
+```
+
+Disable this addon
+```
+vela addon disable chaosblade-operator
+```
+
+## Use
+```shell
+$ vela ls -A | grep chaos
+vela-system     addon-chaosblade-operator       blade-ns                k8s-objects             running healthy                                                               2022-09-14 13:50:16 +0800 CST
+
+$ kubectl get po -A | grep chaos
+blade                          chaosblade-operator-b6f48fdbc-m9ff9                 1/1     Running            0                 3m52s
+blade                          chaosblade-tool-sq7pj                               1/1     Running            0                 3m52s
+```
+
+```shell
+Please access addon-chaosblade-operator from the following endpoints:low (timeout 0/600 seconds)...
++---------+------------+-----------------------------------------+-----------------------------------------+-------+
+| CLUSTER | COMPONENT  |        REF(KIND/NAMESPACE/NAME)         |                ENDPOINT                 | INNER |
++---------+------------+-----------------------------------------+-----------------------------------------+-------+
+| local   | blade-helm | Service/blade/chaosblade-webhook-server | https://chaosblade-webhook-server.blade | true  |
++---------+------------+-----------------------------------------+-----------------------------------------+-------+
+
+```
+
+```shell
+cat <<EOF | kubectl up -f -
+apiVersion: chaosblade.io/v1alpha1
+kind: ChaosBlade
+metadata:
+  name: delete-pod-by-names
+spec:
+  experiments:
+  - scope: pod
+    target: pod
+    action: delete
+    desc: "delete pod by names"
+    matchers:
+    - name: names
+      value:
+      - "kubevela-vela-core-8699fb6d68-mrfvv"
+    - name: namespace
+      value:
+      - "vela-system"
+EOF
```

Comment:
can it be a trait, when this trait added, than the app( with pod detected) will automatically be injected a chaos and get the result integrating with the Grafana addon.